### PR TITLE
RED-138: Solving Phase Problem

### PR DESCRIPTION
**Pull request by jesi-rgb**
# Scene

@nipunramk this would be the whole scene! lemme know your thoughts!

https://user-images.githubusercontent.com/50735312/205651422-e967d629-37c3-4fba-bbc2-91c5bb1384fc.mp4

### CODE REVIEW

**Comment by nipunramk** [comment]
0:52 make sure we plot the circle in a clockwise direction since that is the convention you chose. 

Also adding a rectangle tracking the x and y coordinates would be super cool.

**Comment by nipunramk** [comment]
1:08 -- let's make the vector rotate around the circle to show that it stays the same. Let have the text be $r = \sqrt{x^2 + y^2}$

**Comment by nipunramk** [comment]
@1:24 can we add the sin and cosine versions of the transforms in addition to the full DFT -- that is a very important connection that we should highlight

**Comment by jesi-rgb** [comment]
Done, @nipunramk! whatcha think

https://user-images.githubusercontent.com/50735312/207032788-78fabb69-b7cc-4b78-ac82-17d7b958b49a.mp4

**Comment by jesi-rgb** [comment]
aaaaand more feedback! @nipunramk 

it is kinda fun and goofy to see them bounce around like that haha i enjoy it a lot


https://user-images.githubusercontent.com/50735312/207039564-86141f9a-3c82-4e2e-80fd-256fa9995f83.mp4

File: common/classes.py [hunk 1]
```diff
@@ -368,8 +368,8 @@ def __init__(
 
         super().__init__(
             values,
-            height=height,
-            width=width,
+            # height=height,
+            # width=width,
```
Comment (nipunramk) [inline common/classes.py:372]: @jesi-rgb why is this commented out? I don't see how this is relevant to the rest of the scene?

**Pull request merged**